src/DebuggerData.h

```diff
@@ -19,22 +20,49 @@ struct MemoryLayout
 	int mapperSize[4][4];
 };
 
+struct AddressRange {
+	explicit AddressRange() : start(0) {}
```

Comment:
This is not wrong, but unless there's a good reason to have this, I wouldn't add it. What's special about a range with single address=0?

I quickly checked the current code, but I couldn't see any places where a default-constructed AddressRange is needed. And if there were such places then I'd still prefer to write those as: `AddressRange myRange(/*dummy*/ 0);`

A good result for the default constructor would be an invalid range, but the current code represents invalid ranges via an empty optional (I mean the AddressRange object itself cannot represent an invalid range).

Reply:
That one slipped from before I turned range into an std::optional.